### PR TITLE
[OPPRO-270] improve Velox backend doc

### PR DESCRIPTION
**Pull request by rui-mo**
## What changes were proposed in this pull request?

Added ninja installation command and java requirement.

## How was this patch tested?

pass CI

### CODE REVIEW

**Review by JkSelf — APPROVED** [review]
LGTM

File: docs/Velox.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+Velox uses the script setup-ubuntu.sh to install all dependency libraries, but Arrow's dependency libraries can't be installed.
+Velox also requires ninja for compilation. So we need to install them manually:
 
 ```shell script
-apt install maven build-essential cmake libssl-dev libre2-dev libcurl4-openssl-dev clang lldb lld libz-dev git
+apt install maven build-essential cmake libssl-dev libre2-dev libcurl4-openssl-dev clang lldb lld libz-dev git ninja-build
 ```
 
-Also we need to setup the JAVA_HOME env.
+Also, we need to set up the JAVA_HOME env. Currently, java8 is required.
```
Comment (zhouyuan) [inline docs/Velox.md:14]: maybe also mention java 11/17 support not ready?

Comment (rui-mo) [reply docs/Velox.md]: Added.

**Review by zhouyuan — APPROVED** [review]

**Comment by zhouyuan** [comment]
The GHA failures are not related

**Pull request merged**